<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startPingButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>3</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QComboBox" name="interfaceComboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>2</y>
      <width>410</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="ipAddressComboBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>2</y>
      <width>410</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="targetIPPart1">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>46</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="targetIPPart2">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>46</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="targetIPPart3">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>46</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="targetIPPart4">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>46</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>46</y>
      <width>10</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>46</y>
      <width>10</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>46</y>
      <width>10</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="addTargetButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>46</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>添加IP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addAllTargetButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>46</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>添加全网段</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton1Time">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>46</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>实时延迟</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton3Time">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>46</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>三次平均</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton5Time">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>46</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>五次平均</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="pingResultArea">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>90</y>
      <width>951</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>851</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>任何问题，或建议，请联系作者：韩奥hausa/13598131990 （点击隐藏这句话）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteAllTargetButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>46</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>删除所有</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>500</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>置顶窗口</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
